bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/firmware/Firmware.java

```diff
@@ -376,7 +386,8 @@ private boolean isInt(String s) {
      */
     public static final class Builder {
```

Comment:
Please factor this out into a dedicated `FirmwareBuilder` class, so that it gets symmetric to all the other builders we have.